apps/dashboard/test/integration/dashboard_controller_test.rb

```diff
@@ -13,7 +13,7 @@ def teardown
   end
 
   test "should create Jobs dropdown" do
-    get :index
+    get '/'
```

Comment:
This may be `root` or `root_path`?

Reply:
I thought it should just go to `dashboard#index` only. I was basing the request off the `config/routes.rb` which just gave `root dashboard#index` but I didn't see any entry for `root_path` or `root_url` but I may be mis-understanding the comment.